# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1329774110 28800
#      Mon Feb 20 13:41:50 2012 -0800
# Node ID b5b49f400f9b57deb908324374ef09d76f869b3f
# Parent  966ec84e7858316ecd9732acd1f493ab86dabc7a
__bar__.m: Add missing semicolon to stop internal results being printed.

* __bar__.m: Add missing semicolon to stop internal results being printed.

diff --git a/scripts/plot/private/__bar__.m b/scripts/plot/private/__bar__.m
--- a/scripts/plot/private/__bar__.m
+++ b/scripts/plot/private/__bar__.m
@@ -77,17 +77,19 @@ function varargout = __bar__ (vertical, 
       group = false;
       idx++;
     else
       if ((ischar (varargin{idx}) || iscell (varargin{idx}))
           && ! have_line_spec)
         [linespec, valid] = __pltopt__ (func, varargin{idx}, false);
         if (valid)
           have_line_spec = true;
-          newargs = [{"facecolor", linespec.color}, newargs]
+          ## FIXME: strange parse error requires semicolon to be spaced
+          ##        away from closing ']' on next line.
+          newargs = [{"facecolor", linespec.color}, newargs] ;
           idx++;
           continue;
         endif
       endif
       if (isscalar(varargin{idx}))
         width = varargin{idx++};
       elseif (idx == nargin - 2)
         newargs = [newargs,varargin(idx++)];
